src/test/common/clara/test_accumulation.cljc

```diff
@@ -1540,9 +1551,10 @@
                             (cond (false? a) false
                                   (false? b) true
                                   :else (throw
-                                         (IllegalStateException. (str "This test should only compare false and numeric values, "
-                                                                      \newline not
-                                                                      "a numeric value to another numeric value")))))
+                                         (ex-info (str "This test should only compare false and numeric values, "
+                                                       \newline not
+                                                       "a numeric value to another numeric value")
+                                                  {:a a :b b}))))
```

Comment:
We can't throw IllegalStateException in ClojureScript since it is a JVM class.  This would have failed with another exception if we actually reached this in a ClojureScript/JavaScript environment.